src/lucky/routeable.cr

```diff
@@ -115,4 +115,16 @@ module Lucky::Routeable
       \{% raise "Use `route` instead of `with` if the action doesn't need params" %}
     end
   end
+
+  macro included
```

Comment:
Is this explained anywhere in the crystal docs? I THINK it means "always run this when 'include Routable' is called?"  so we can use it to initialize macro consts etc., but not really sure on the specifics.

Reply:
Here is a nice guide on it. They're called Macro Hooks: https://crystal-lang.org/docs/syntax_and_semantics/macros/hooks.html

It basically copies what's in that macro into the class that is including/inheriting it